### DISCUSSION TITLE
Graph Plotting Problem

### DISCUSSION THREAD

**Discussion by chanemaster** [xournalpp/xournalpp#3153] [Q&A]
**Affects versions :**
 - OS: Windows 10
 - Which version of libgtk do you use: 3.24.29
 - Version of Xournal++: 1.1.0+dev
 - Installation method: xournalpp-1.1.0.dev-nightly.20210609-windows.zip

**Describe the bug**
When using LaTeX to plot graph e.g. \graph{x^2}, the graph will always appear at lower right corner and when I drag it to the center some part of the graph disappear. Moreover, command \graph{x^2} cannot be edited after the graph show up.

I also would like to ask if there is a tool to draw table without typing in LaTeX. For example, just click and drag mouse to draw table.

**To Reproduce**
1. Tools
2. Add/Edit Tex
3. Type command for plotting graph such as \graph{x^2}

**Expected behavior**
I expect the full graph to show at the center with Tex command show under the graph in order to check or correct easily.

**Screenshots of Problem**
![2021-06-10](https://user-images.githubusercontent.com/85364302/121493422-06565a80-ca02-11eb-8844-68e5064b3924.png)
![2021-06-10 (1)](https://user-images.githubusercontent.com/85364302/121493464-10785900-ca02-11eb-9579-3f0548ec5443.png)

**Additional context**
Now I have to press enter to plot graph. After finished, when I double click the graph in order to edit, it always shows at the lower right corner and then I move it to the center to see some part of the graph disappear.(The same screenshot results that I attached for you.)

**Comment by rolandlo** [comment]
The LaTeX preamble you use is not part of the official Xournal++. You can modify it to your needs, for example by restricting the height of the graph. 
Improvements on the resizing of the LaTeX window will be introduced in #2809.
